packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -112,6 +119,15 @@ export const NavigatorWrapper = () => {
     routeNameRef.current = currentRouteName
   }
 
+  let updateRequired = false
+  if (minRequiredVersion) {
+    const version = DeviceInfo.getVersion()
+    updateRequired = isVersionBelowMinimum(version, minRequiredVersion)
+    if (!updateRequired) {
+      dispatch(setRequiredVersion(null))
+    }
+  }
```

Comment:
We shouldn't dispatch during render.
This logic should live in a saga.

Reply:
This is run only once. Which saga do you think this belongs on? I understand where you're coming from, but this seemed like the most intuitive way to solve this and I don't see a big risk dispatching here.